scenarios/action_recognition/README.md

```diff
@@ -1,66 +1,111 @@
-# Human Action Recognition 
+# Human Action Recognition
+
 ```diff
-+ 12/4/2019: This is work-in-progress.
++ Feb 2020: We are working on moving all code from contrib\action_recognition to this folder.
++           While this work is ongoing, please visit both locations for implementations and documentation.
```

Comment:
I believe 'master' branch should only include product-ready / fully functional and fully tested codes (sometimes things can be broken in master though). In that sense, I think we should remove any components that are still work-in-progress from master. Users who want to taste future features always can checkout to staging - or use contrib (that's why we add contrib) - and use them at their own risks.